cosmosdb/mongodb/ps-mongodb-ru-get.ps1

```diff
@@ -1,13 +1,13 @@
 # Get RU for an Azure Cosmos MongoDB API database or collection
 $apiVersion = "2015-04-08"
 $resourceGroupName = "myResourceGroup"
-$databaseThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/databases/settings"
-$collectionThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/databases/collections/settings"
 $accountName = "mycosmosaccount"
 $databaseName = "database1"
 $collectionName = "collection1"
 $databaseThroughputResourceName = $accountName + "/mongodb/" + $databaseName + "/throughput"
+$databaseThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/databases/settings"
 $collectionThroughputResourceName = $accountName + "/mongodb/" + $databaseName + "/" + $collectionName + "/throughput"
+$collectionThroughputResourceType = "Microsoft.DocumentDb/databaseAccounts/apis/databases/collections/settings"
```

Comment:
Is the $collectionThroughputResourceType variable not used in this script? you can delete if not used.

Reply:
Yes, it is used in the second Get operation here.

Get-AzResource -ResourceType $collectionThroughputResourceType `
    -ApiVersion $apiVersion -ResourceGroupName $resourceGroupName `
    -Name $collectionThroughputResourceName  | Select-Object Properties